src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -378,4 +483,45 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
       });
     }
   }
+
+  enableRgwModule() {
+    let $obs;
+    const fnWaitUntilReconnected = () => {
+      observableTimer(2000).subscribe(() => {
+        // Trigger an API request to check if the connection is
+        // re-established.
+        this.mgrModuleService.list().subscribe(
+          () => {
+            // Resume showing the notification toasties.
+            this.notificationService.suspendToasties(false);
+            // Unblock the whole UI.
+            this.blockUI.stop();
+            // Reload the data table content.
+          },
```

Comment:
The notification toast should be put here because this is where the toasties are unblocked. Hence the reload should also go here instead of a manual timeout!
```suggestion
            this.blockUI.stop();
            // Reload the data table content.
            this.notificationService.show(NotificationType.success, $localize`Enabled RGW Module`);
            this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
              this.router.navigate(['/rgw/multisite']);
            }); 
          },
```

Reply:
Updated, Thanks